tests/instancetype/instancetype.go

```diff
@@ -297,7 +303,7 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			preference.Spec.Firmware = &instancetypev1beta1.FirmwarePreferences{
 				PreferredUseBios: pointer.P(true),
 			}
-			preference.Spec.PreferredTerminationGracePeriodSeconds = pointer.P(int64(15))
+			preference.Spec.PreferredTerminationGracePeriodSeconds = pointer.P(int64(preferredTerminationGracePeriodSeconds))
```

Comment:
KubeVirt has a generic pointer package:
https://github.com/kubevirt/kubevirt/blob/43531861ac5fc7795ac6523c482b2e2de74dfa3b/pkg/pointer/pointer.go#L22

Please consider using it.